docs/development/getting_started.rst

```diff
@@ -0,0 +1,66 @@
+===============
+Getting started
+===============
+
+
+.. contents:: :local:
+
+
+General
+-------
+
+- the minimal python version is Python 3.6.0, don't break it,
```

Comment:
I guess you forgot to add that this version is *required*

Reply:
It's in development section, not run requirements. I wanted to highlight minimal version that we want to support. So PRs that use 3.7 syntax are discouraged.

Reply:
You meant 3.5? As far as I know, 3.7 does not provide any breaking syntax changes comparing to 3.6

Reply:
you know nothing ;) https://docs.python.org/3.7/whatsnew/3.7.html

So for example use of `breakpoint` will break in 3.6. Dataclasses and so on ... and I know it isn't strictly syntax I meant any functionality.